www/js/yodaAnswerGetter.js

```diff
@@ -40,6 +38,52 @@ $(function () {
     setInterval(getAnsweredJson, 2900);
 });
 
+$(document).bind('pageinit', function () {
+    $('#search').keypress(function(event) {
+        var keycode = (event.keyCode ? event.keyCode : event.which);
+        if(keycode == '13'){
+            submit();
+        }});
+    $('#search').val(question);
+})
+
+function submit(){
+    $.ajax({
+        type: "POST",
+        url: CONNECTION_ADDRESS+"q",
+        data: getDataToSend(),
+        success: function (response) {
+            $('#verticalCenter').animate({marginTop: '0px'}, 'slow');
+            switchToSearchAfterAnswer();
+            setTimeout(function () {
+                loadQuestion(JSON.parse(response).id, true)
+            }, 500);
+        }
+    });
+}
+
+function getDataToSend(){
+    var data={};
+    data.question=$("#search").val();
+    var numberOfConcepts=$("#numberOfConcepts").val();
+    var i=1;
+    for(;i<=numberOfConcepts;i++){
+        data['fullLabel' + i]=$("#fullLabel"+i).val();
+        data['pageID' + i]=$("#pageID"+i).val();
+    }
+    if (generatedConcepts!=null){
+        for(var j=0;j<generatedConcepts.length;j++){
+            if (conceptButtons[j+1]==SELECTED){
+            data['fullLabel' + i]=generatedConcepts[j].title;
+            data['pageID' + i]=generatedConcepts[j].pageId;
+            i++;
+            }
+        }
+    }
+    data.numberOfConcepts= i;
+    return data;
+}
```

Comment:
@ermrk (I know we talked about it briefly, but at that point I thought it's about sending json rather than query data. Doesn't seem to be the case though, right?)